docs/spelling_wordlist

```diff
@@ -10,6 +10,7 @@ affordances
 aggregator
 Ai
 Alchin
+allowlist
```

Comment:
This is unused.

Reply:
OK - I've removed these, also added whitelist for now (can be removed with email validator change)